Doc/library/sqlite3.rst

```diff
@@ -1088,6 +1102,60 @@ Exceptions
    transactions turned off.  It is a subclass of :exc:`DatabaseError`.
 
 
+.. _sqlite3-blob-objects:
+
+Blob Objects
+------------
+
+.. versionadded:: 3.11
+
+.. class:: Blob
+
+   A :class:`Blob` instance is a :term:`file-like object` that can read and write
+   data in an SQLite :abbr:`BLOB (Binary Large OBject)`.
+
+   .. method:: Blob.close()
+
+      Close the blob.
+
+      The blob will be unusable from this point onward.  An
+      :class:`~sqlite3.Error` (or subclass) exception will be raised if any
+      further operation is attempted with the blob.
+
+   .. method:: Blob.__len__()
+
+      Return the blob size in bytes.
+
+   .. method:: Blob.read(length=-1, /)
+
+      Read *length* bytes of data from the blob at the current offset position.
+      If the end of the blob is reached, the data up to end of file will be
```

Comment:
Not 100% sure on this as I know EOF is a common acronym; feel free to ignore if you disagree!

```suggestion
      If the end of the blob is reached, the data up to the end of the file will be
```

Reply:
Maybe it's better to use the acronym :)

```console
$ grep -r EOF Doc | wc -l
     110
$ grep -ri "end.of.file" Doc | wc -l
      21
```

Reply:
FTR: `grep -r "abbr.*EOF" Doc` gives no hits.

Reply:
How about 8d834bf0d5b7d48a41648ab4b72b006738a926e8?